### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by n2ygk** [django-oauth/django-oauth-toolkit#1525] [Q&A]
Seeing if I non-admins can post here.

**Comment by n2ygk-demo** [comment]


**Reply by n2ygk** [reply]
I've opened an issue with Jazzband roadies to figure out the permissions issue.

**Comment by catdevnull** [comment]
i basically have two questions:
1. how can I find an AccessToken’s Grant? they don’t seem to be linked
2. what’s the correct way to add an extra step to authorization? in our case, we want the user to select one of their existing organizations to link the Grant to.
